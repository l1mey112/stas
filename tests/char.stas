include <std.stas>

fn main {
	`A`  putuln
	`B`  putuln
	`\n` putuln
	`\\` putuln
	`\\` putuln
	` `  putuln
	`a`  putuln

	0
	while dup 256 < {
		dup dup `0` >= swap `9` <= & if {
			dup putc
		}
		dup dup `a` >= swap `z` <= & if {
			dup putc
		}
		dup dup `A` >= swap `Z` <= & if {
			dup putc
		}
		++
	}
	drop
}