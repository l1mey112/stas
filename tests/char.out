65
66
10
92
92
32
97
0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz